commit 02987888adc8575cd8efa103fe901de69224f4b4
Author: Vincent Bernat <bernat@luffy.cx>
Date:   Thu Feb 20 21:57:29 2014 +0100

    lldpcli: fix readline() wrapper to take prompt as argument
    
    `readline()` is expecting the prompt as argument. We fix the wrapper to
    have the same requirement.

diff --git a/src/client/lldpcli.c b/src/client/lldpcli.c
index 4223c90..35c1b94 100644
--- a/src/client/lldpcli.c
+++ b/src/client/lldpcli.c
@@ -239,10 +239,10 @@ cmd_help(int count, int ch)
 }
 #else
 static char*
-readline()
+readline(const char *p)
 {
 	static char line[2048];
-	fprintf(stderr, "%s", prompt());
+	fprintf(stderr, "%s", p);
 	fflush(stderr);
 	if (fgets(line, sizeof(line) - 2, stdin) == NULL)
 		return NULL;
